Electrical/BMS/Startup.md

```diff
@@ -1 +1,14 @@
 # Startup Board
+As previously discussed, the BMS, whether the Orion BMS or an eventual custom BMS we may eventually build, is responsible for taking various inputs from our battery pack, and converting those into signals we can use to determine whether it's safe to start the vehicle. The startup board uses the Discharge Enable (DE), Multipurpose Enable (ME), Charge Enable (CE), and Charge Safety Enable (CSE), as well as 12 Main (12M_L, active low signal) and 12 Trip (12T), which don't come from the BMS. To get a better picture of how the BMS operates, refer to the Orion BMS section. To gain an understanding of the other signals Startup doesn't use, refer to the Orion BMS Wiring manual provided on that page.
+## The BMS Signals
+The four signals provided by the BMS may be self-explanatory, but it's important to have a solid understanding of what they mean. The BMS signals are all open-drain, meaning when true the signal sinks current to ground when false, has high impedance. DE is an open-drain signal that pulls down to ground when the battery is able to be discharged. The ME does the same thing, though the Orion can be programmed to turn this signal on based on other user-defined criteria. The two charge signals are used to determine whether it's safe to charge, and the startupt board will connect the solar array's MPPT to the battery pack if they're both true. CE and CSE are again an open drain output, and CE is true when the batteries are adequately balanced, not full, and not too hot. The CSE signal is a little less useful for our application, though similar to the ME, we use the CSE as a backup signal to stop vehicle charging should conditions be inadequate. In many traditional electric vehicle applications, the vehicle can get charge through two sources, either a wall outlet or regenerative braking. CSE is primarily meant to allow the BMS to shut off wall outlet charging should the analog on/off switch fail.
+## The Other Signals
+In addition to the 4 BMS sourced signals, there's also 12 Main and 12 Trip. 12 Trip represents the supplemental power supplied by a small lead acid battery that allows the vehicle to always be connected to some sort of power. 12 Main represents the main power coming from the battery pack, and this signal is high when the battery is connected to the vehicle. 
+## Startup Circuit
+### Schematic
+Here's a schematic of the current Startup Circuit:
+![Startup](Electrical/BMS/Startup Board.jpg)
```

Comment:
Put the image in the `_static` folder and update the URL. See example here: https://raw.githubusercontent.com/Solar-Gators/docs/master/Electrical/Telemetry/gui.md

Reply:
`![GUI Preview](/_static/telemetry/live-telemetry.png)`